### PR TITLE
DIANA-12 Create ReaderField implementations for java.time.LocalDate

### LINKED ISSUE #12: Create ReaderField implementations for java.time.LocalDate

### PR DESCRIPTION
**Pull request by danielsoro**
fixes #12

### CODE REVIEW

File: diana-api/src/main/java/org/jnosql/diana/api/reader/LocalDateReader.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    public <T> T read(Class<T> clazz, Object value) {
+
+        if (LocalDate.class.isInstance(value)) {
+            return (T) value;
+        }
+
+        if (Number.class.isInstance(value)) {
+            return (T) new Date(((Number) value).longValue()).toInstant().atZone(ZoneId.systemDefault()).toLocalDate();
+        }
+
```
Comment (otaviojava) [inline diana-api/src/main/java/org/jnosql/diana/api/reader/LocalDateReader.java]: @danielsoro what do you think to put conditions to `java.util.Date` and `java.util.Calender`

File: diana-api/src/main/java/org/jnosql/diana/api/reader/LocalDateReader.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        return (T) new Date(value.toString()).toInstant().atZone(ZoneId.systemDefault()).toLocalDate();
```
Comment (otaviojava) [inline diana-api/src/main/java/org/jnosql/diana/api/reader/LocalDateReader.java]: @danielsoro WDYT if we use `LocalDate.parse` instead?:
https://docs.oracle.com/javase/8/docs/api/java/time/LocalDateTime.html#parse-java.lang.CharSequence-

File: diana-api/src/main/java/org/jnosql/diana/api/reader/LocalDateReader.java (hunk 1 above)
Comment (danielsoro) [inline diana-api/src/main/java/org/jnosql/diana/api/reader/LocalDateReader.java]: Make sense! :)
I'll update it.

File: diana-api/src/main/java/org/jnosql/diana/api/reader/LocalDateReader.java (hunk 2 above)
Comment (danielsoro) [inline diana-api/src/main/java/org/jnosql/diana/api/reader/LocalDateReader.java]: +1

**Pull request merged**